### DISCUSSION TITLE
Install with antigen failed

### DISCUSSION THREAD

**Discussion by yueyejinghun** [marlonrichert/zsh-autocomplete#350] [Q&A] [answered]
* `zsh-autocomplete` version: 817af7c
* Zsh version: 5.8 on Fedora (plus Ubuntu 20.04 and Arch)
* Framework:  Oh My Zsh
* Plugin manager:  antigen (v2.2.2. also tested with antigen nightly build revision date 2018-01-15)
* Error message: 
```zsh
Installing marlonrichert/zsh-autocomplete... Error! Activate logging and try again
```
* Content of .zshrc: 

```zsh
source ~/antigen.zsh
antigen bundle marlonrichert/zsh-autocomplete
antigen apply
```
This issue started to appear in the middle or late February if I remember it right. The first time I installed this plugin with antigen is Jan. 14 because on that day I saw [this](https://stackoverflow.com/a/65461206) answer on StackOverflow. Antigen had no problem install this plugin then. Due to antigen cannot clone a specific branch of a GitHub repo, therefore I am unable to test which branch started to cause this. Now I can manually clone the repo to $HOME/.antigen/bundles/marlonrichert/zsh-autocomplete and it works fine.
[debug_ac_only.log](https://github.com/marlonrichert/zsh-autocomplete/files/6073293/debug_ac_only.log)

**Comment by felipecrs** [comment, marked as answer]
Simply change your `.zshrc`:

```diff
source ~/antigen.zsh
- antigen bundle marlonrichert/zsh-autocomplete
+ antigen bundle marlonrichert/zsh-autocomplete@main
antigen apply
```

And it will work.
